### ISSUE TITLE
Allow configuring TLS on the Tasklist ingress

### ISSUE THREAD

**Issue by npepinpe** [camunda/camunda-platform-helm#360]
Every chart with an ingress templates allows configuring TLS in the same way, except Tasklist. We should have the same values so that TLS can be terminated at the ingress.

Expected values in the `ingress.yaml` file:

```yaml
  {{- if .Values.ingress.tls.enabled }}
  tls:
    - hosts:
        - {{ .Values.ingress.host }}
      {{- if .Values.ingress.tls.secretName }}
      secretName: {{ .Values.ingress.tls.secretName }}
      {{- end }}
  {{- end }}
```

See any of the other charts as an example.

**Comment by conadrisca** [comment]
+1 Here but would also like to point out that this is missing from the optimize helm charts also.

**Optimize**: https://github.com/camunda/camunda-platform-helm/blob/camunda-platform-8.0.11/charts/camunda-platform/charts/optimize/templates/ingress.yaml
**Tasklist**: https://github.com/camunda/camunda-platform-helm/blob/camunda-platform-8.0.11/charts/camunda-platform/charts/tasklist/templates/ingress.yaml

**Comment by npepinpe** [comment]
The Optimize one was merged recently but not yet released. It will be in the next version soon.

**Comment by conadrisca** [comment]
PR for this feature:
https://github.com/camunda/camunda-platform-helm/pull/390

**Comment by aabouzaid** [comment]
Fixed by: #390
